deeplabcut/pose_estimation_tensorflow/predict_videos.py

```diff
@@ -1297,6 +1298,10 @@ def convert_detections2tracklets(
         )
 
     cfg = auxiliaryfunctions.read_config(config)
+    if len(cfg["multianimalbodyparts"]) == 1 and track_method != "box":
+        warnings.warn("Switching to `box` tracker for single point tracking...")
+        track_method = "box"
```

Comment:
maybe this conflicts with pending merge of track goes to config file?

Reply:
Yeah, let's fix it in the other PR (which we'll merge afterwards)

Reply:
@jeylau Thanks for your work !

Hey @MMathisLab  ill review it by tomorrow and let you know.